backend/utils/room_utils.py

```diff
@@ -118,3 +120,19 @@ async def is_user_starred_room(org_id: str, room_id: str, member_id: str) -> boo
     if response and "status_code" not in response:
         return response["room_members"][member_id]["starred"]
     raise Exception("Room not found")
+
+
+async def remove_member(room_obj: dict, mem_id: str, org_id: str):
```

Comment:
Can this remove_member function return a boolean.
If the update method is successful, let it return true,and if it fails
Let it return false.

What do you think ?